hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/TestS3AAWSCredentialsProvider.java

```diff
@@ -235,7 +237,7 @@ public void testAssumedRoleWithRemap() throws Throwable {
     Assertions
         .assertThat(credentials.size())
         .describedAs("List of Credentials providers")
-        .isEqualTo(3);
+        .isEqualTo(TERMINATION_TIMEOUT);
```

Comment:
why is `credentials.size()` equal to `TERMINATION_TIMEOUT`?

Reply:
ooh, good catch. I pulled out the constant and must have found that as a match. will fix